tests/shell/BashateBearTest.py

```diff
@@ -0,0 +1,28 @@
+from bears.shell.BashateBear import BashateBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+E002_bad = """
+  echo "E002: Has a tab"
+"""
+
+E003_bad = """
+   indent is not multiple of 4
+"""
+long_line = 'a = "{0}"'.format('a' * 100)
+
```

Comment:
other tests add another blank line, would recommend that too to match the style.

Reply:
you want me to add a newline character after the braces?

Reply:
no just at this location ;)